content/uk/docs/reference/glossary/deployment.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Deployment
+id: deployment
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/deployment/
+# short_description: >
+#  An API object that manages a replicated application.
+short_description: >
+  Об'єкт API, що керує реплікованим застосунком.
+
+aka:
+tags:
+- fundamental
+- core-object
+- workload
+---
+<!-- An API object that manages a replicated application. -->
+Об'єкт API, що керує реплікованим застосунком.
+
+<!--more-->
+
+<!-- Each replica is represented by a {{< glossary_tooltip term_id="pod" >}}, and the Pods are distributed among the nodes of a cluster. -->
+Кожна репліка являє собою {{< glossary_tooltip term_id="pod" text="Под" >}}; Поди розподіляються між вузлами кластера.
```

Comment:
nit:
```suggestion
Кожна репліка являє собою {{< glossary_tooltip term_id="pod" text="Pod" >}}; Поди розподіляються між вузлами кластера.
```
?

Reply:
Same to https://github.com/kubernetes/website/pull/20020#discussion_r403159215